drivers/hmis/app/graphql/types/hmis_schema/geolocation_coordinates.rb

```diff
@@ -0,0 +1,16 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::GeolocationCoordinates < Types::BaseObject
+    field :id, ID, null: false
+    field :latitude, String, null: true, method: :lat
+    field :longitude, String, null: true, method: :lon
```

Comment:
No they shouldn't, but the db table has them as nullable. I will update this to make them non-nullable and use a scope instead to filter out any geolocations that are missing lat/lon, since we can't show them anyway.

Reply:
Done with https://github.com/greenriver/hmis-warehouse/pull/4993/commits/9be98fd44e973408ebaf9890747742c551e8341d